sqldb/sqlc/queries/graph.sql

```diff
@@ -565,6 +565,14 @@ JOIN channel_policy_extra_types cpet
 ON cp.id = cpet.channel_policy_id
 WHERE cp.id = $1 OR cp.id = $2;
 
+-- name: GetV1DisabledSCIDs :many
```

Comment:
What's the logic that makes this V1 specific? Won't this work the same way for V2?

Reply:
added a comment 👍

Reply:
also just added a `version = 1` clause